### PR TITLE
Two admin portal bug fixes - 1115 and 967

### PR DESCRIPTION
**Pull request by BradHN1**
####  Summary / Highlights
This PR should have been to Dev but I accidentally committed these fixes to the development branch.

#1115 Unable to create team
#967 A minor typo.

These fixes should get deployed to Dev (update version number to 4.13.6 and tag as dev@4.13.6) and then be merged into Canary and then to Prod when that is deployed.

#### Details (Give details about what this PR accomplishes, include any screenshots etc)
The creating team bug was an undefined variable (isSuperAdmin). 

#### Testing Steps (Provide details on how your changes can be tested)
Create a team in the admin portal.  Should not give a blank screen.

#### Requirements (place an `x` in each `[ ]`)
* [ ] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [ ] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [ ] I've given my PR a meaningful title.
* [ ] I linked this PR to the relevant issue.
* [ ] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

**Review by archx3 — APPROVED** [review]
Great job!

File: app/containers/MassEnergizeSuperAdmin/Teams/CreateNewTeamForm.js [hunk 1]
```diff
@@ -43,11 +43,14 @@ const styles = (theme) => ({
  }) {
   const [parents, setParents] = React.useState([]);
 
+  const isSuperAdmin = auth?.is_super_admin;
```
Comment (Opoku-Agyemang) [inline app/containers/MassEnergizeSuperAdmin/Teams/CreateNewTeamForm.js:46]: what happens if isSuperAdmin is null?  or undefined?

Comment (frimpongopoku) [reply app/containers/MassEnergizeSuperAdmin/Teams/CreateNewTeamForm.js:46]: @BradHN1 your code still breaks. You did not import "auth" as a prop before using it inside "CreateNewTeamForm".

**Review by frimpongopoku — CHANGES_REQUESTED** [review]

**Comment by BradHN1** [comment]
@frimpongopoku - sorry I thought I had tested it but didn’t remember.  Could you fix it and also #1118 ?

**Comment by frimpongopoku** [comment]
@abdullai-t fixed this in another PR 🙏🏾

**Pull request closed without merging**